tests/reconnect_test.cpp

```diff
@@ -50,26 +42,44 @@ static server_config default_value()
 		("nonblocking_io_thread_num", 1)
 		("net_thread_num", 1)
 		("caches_number", 1)
+		("reconnect_parallelism", reconnect_parallelism)
 	;
 
-	server.backends[0]("enable", true)("group", 1);
+	server.backends[0]("enable", true)("group", static_cast<int>(server_index + 1));
```

Comment:
Why isn't `server_index` just an `int`?

Reply:
Typically `size_t` or `ptrdiff_t` is used for indexes, size of these types depends on 32/64-bit architecture, and `size_t` is compatible with standard containers's `.size()`/`operator[]`.

Yes, we can use `int` for simplicity, and I see it's done so everywhere in C code. But in C++ it seems like carelessness or laziness.